### PR TITLE
[Refactor] Final UI tests updates

### PR DESCRIPTION
**Pull request by vhuseinova-msft**
## Purpose

* Improved code style
* Updated setup test

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

## Pull Request Type
What kind of change does this Pull Request introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [x] Refactoring (no functional changes, no api changes)
- [ ] Documentation content changes
- [ ] Other... Please describe:

### CODE REVIEW

**Comment by vhuseinova-msft** [comment]
`try catch` changed to `try?`:
1) for now there is no additional logic that should be handled in `FloatingUITestWindow` by catching errors and in case of errors, UI tests will fail as expected
2) it looks like addtional printing is not a usuful tool here as it's not added to Debug output and/or to AppCenter logs.

**Review by emlynmac — DISMISSED** [review]

**Review by jimchou-dev — DISMISSED** [review]
LGTM

**Review by jimchou-dev — APPROVED** [review]

File: AzureCommunicationUI/AzureCommunicationUIDemoApp/Sources/Views/FloatingUITestWindow.swift [hunk 1]
```diff
@@ -86,100 +86,70 @@ class FloatingUITestWindow: UIWindow {
     @objc func holdButtonTapped(sender: UIButton) {
         debugPrint("UI Test:: holdButtonTapped")
         Task {
-            do {
-                try await callingSDKWrapperMock?.holdCall()
-            } catch {
-            }
+            try? await callingSDKWrapperMock?.holdCall()
```
Comment (JoshuaLai) [inline AzureCommunicationUI/AzureCommunicationUIDemoApp/Sources/Views/FloatingUITestWindow.swift:89]: Do we not want to catch and log the error if something goes wrong?

File: eng/pipelines/jobs/templates/set-appcenter-devices.yml [hunk 2]
```diff
@@ -107,7 +107,7 @@ steps:
               APPCENTER_DEVICES=iphonexrset
               ;;
             04)
-              APPCENTER_DEVICES=iphonexrmaxset
+              APPCENTER_DEVICES=iphonexsmaxset
```
Comment (JoshuaLai) [inline eng/pipelines/jobs/templates/set-appcenter-devices.yml:110]: Was this change intentional? I remember that this was based on a set schedule so did the schedule change?

**Review by JoshuaLai — APPROVED** [review]
Just have a few questions otherwise the changes look fine

File: AzureCommunicationUI/AzureCommunicationUIDemoApp/Sources/Views/FloatingUITestWindow.swift (hunk 1 above)
Comment (vhuseinova-msft) [reply AzureCommunicationUI/AzureCommunicationUIDemoApp/Sources/Views/FloatingUITestWindow.swift:89]: not now. There aren't any errors for `callingSDKWrapperMock ` functions that used in `FloatingUITestWindow` for now. I don't want to add `fatalError` inside catch, as a default behaviour, because in this case error might not be fatal (as it's for UI tests). Also, it looks like addtional printing is not a usuful tool here as it's not added to Debug output and/or to AppCenter logs.
In case if some functions will be updated with throwing error, it would mean that it was done to be handled in UI tests. So the author can add additional logic for error in  that specific case.

File: eng/pipelines/jobs/templates/set-appcenter-devices.yml (hunk 2 above)
Comment (vhuseinova-msft) [reply eng/pipelines/jobs/templates/set-appcenter-devices.yml:110]: we don't have iphonexrmaxset set and because of this 1 of the today's test failed
